alpakaConfig.cmake

```diff
@@ -364,6 +364,12 @@ IF(ALPAKA_ACC_GPU_CUDA_ENABLE)
                 ENDIF()
 
             ELSE()
+                # CUDA 9.0 removed the __CUDACC_VER__ macro. Boost versions lower than 1.65.1 still use this macro.
+                IF(CUDA_VERSION VERSION_GREATER 9.0 AND Boost_VERSION VERSION_LESS 1.65.1)
```

Comment:
Shouldn't this be >= 9.0? This will not catch CUDA 9.0

Reply:
You are right, this was a mistake...